utils/cloudWatchEvent.js

```diff
@@ -0,0 +1,68 @@
+const {
+    PutRuleCommand,
+    PutTargetsCommand,
+} = require('@aws-sdk/client-eventbridge');
+
+const { addPermissions } = require('./addPermissions');
+const { CLOUDWATCH_EVENT } = require('./constants');
+
+const { EventBridgeClient } = require('@aws-sdk/client-eventbridge');
+
+exports.cloudWatchEvent = async (
+    name,
+    rangeTime,
+    region,
+    accessKey,
+    secretKey,
+    iamRoleArnEvent,
+    accountId,
```

Comment:
Why does the user need to provide the `account id` if we have `accessKey` and `secretKey` cant we just get the account id from the AWS cli?

Reply:
Yes will add a function to receive account id